deis-cleanup.rb

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env ruby
+
+require "open3"
+require "csv"
+
+KEEP_LAST_VERSIONS = ENV["KEEP_LAST_VERSIONS"].to_i || 2
+DRY_RUN = ENV["DRY_RUN"] || false
+EXCLUDE_REGEX = case ENV["EXCLUDE_REGEX"]
+                when nil then /alpine|deis|blackhole|none|datadog|cleanup|heroku|python/
+                else /#{ENV["EXCLUDE_REGEX"]}/
+                end
```

Comment:
You don't need a `case` construction here. All you want to do is take a value from the environment variable if present or use a default one when not.